test/Bridges/Variable/zeros.jl

```diff
@@ -127,17 +127,13 @@ function test_zeros()
     @test MOI.get(bridged_mock, MOI.ConstraintDual(), cx) == 0.0
     @test MOI.get(bridged_mock, MOI.ConstraintDual(), c1) == 0.0
     @test MOI.get(bridged_mock, MOI.ConstraintDual(), c2) == 1.0
-    bridge = MathOptInterface.Bridges.Variable.ZerosBridge{Float64}
     attr = MOI.ConstraintDual()
-    err = ArgumentError(
-        "Bridge of type `$(bridge)` does not support accessing " *
-        "the attribute `$attr`. If you encountered this error " *
-        "unexpectedly, it probably means your model has been " *
-        "reformulated using the bridge, and you are attempting to query " *
-        "an attribute that we haven't implemented yet for this bridge. " *
-        "Please open an issue at https://github.com/jump-dev/MathOptInterface.jl/issues/new " *
-        "and provide a reproducible example explaining what you were " *
-        "trying to do.",
+    err = ErrorException(
+        "Unable to query the dual of a variable bound that was reformulated " *
+        "using `ZerosBridge`. This usually arises in conic models when a " *
```

Comment:
Why conic models ? Because conic solvers do not implement variable bounds ?

Reply:
I put "usually" because most people are going to hit this in JuMP using a conic solver like SCS and `p == 1`. Even if it can happen in other cases, making the error more explanatory for the common case seems useful.

Reply:
Then I would say `conic solvers`, not conic model. If you solve an LP or QP with SCS, you are going to get this error.